l1-contracts/contracts/bridge/ntv/L2NativeTokenVault.sol

```diff
@@ -125,13 +125,34 @@ contract L2NativeTokenVault is IL2NativeTokenVault, NativeTokenVault {
         }
     }
 
-    /// @notice Deploys the beacon proxy for the L2 token, while using ContractDeployer system contract.
+    /// @notice Ensures that the token is deployed inner for legacy tokens. 
+    function _ensureTokenDeployedInnerLegacyToken(
```

Comment:
No, [ensure](https://www.google.com/search?q=ensure+meaning&sca_esv=be665a62fa86d23b&ei=AFUOZ7KlI9SmhbIPotiDuQ8&ved=0ahUKEwiypaCEp5CJAxVUU0EAHSLsIPcQ4dUDCA8&uact=5&oq=ensure+meaning&gs_lp=Egxnd3Mtd2l6LXNlcnAiDmVuc3VyZSBtZWFuaW5nMhYQABiABBiRAhixAxiDARiKBRhGGPkBMgsQABiABBiRAhiKBTILEAAYgAQYkQIYigUyBRAAGIAEMgUQABiABDIFEAAYgAQyBRAAGIAEMgUQABiABDIFEAAYgAQyBRAAGIAEMjAQABiABBiRAhixAxiDARiKBRhGGPkBGJcFGIwFGN0EGEYY-QEY9AMY9QMY9gPYAQFIlghQ9gFYnwdwAXgBkAEAmAFooAHSBaoBAzUuM7gBA8gBAPgBAZgCCaAC-QXCAgoQABiwAxjWBBhHwgINEAAYgAQYsAMYQxiKBcICEBAAGIAEGLEDGEMYgwEYigXCAggQABiABBixA8ICCxAAGIAEGLEDGIMBwgIREAAYgAQYkQIYsQMYgwEYigXCAjAQABiABBiRAhixAxiDARiKBRhGGPkBGJcFGIwFGN0EGEYY-QEY9AMY9QMY9gPYAQGYAwCIBgGQBgq6BgYIARABGBOSBwM2LjOgB71Q&sclient=gws-wiz-serp) has an active element to it, it can do something: 
mak

> e sure
> make certain
> see to it
> secure
>

Reply:
"Make sure" sounds good, but still it doesn't sound like saving something

Reply:
Then we should rename all ensureTokenDeployed functions?

Reply:
Not a priority, but would be nice imho